redirect-handler.js

```diff
@@ -18,6 +18,16 @@ const yaml = require('js-yaml');
 const fs = require('fs');
 const escapeStringRegexp = require('escape-string-regexp');
 
+const baseUrlPrefix = 'https://web.dev/';
+const baseUrl = new URL(baseUrlPrefix);
+
+if (!baseUrlPrefix.endsWith('/')) {
+  // Prevents abuse by having a domain "https://web.dev.another.owner.com".
+  throw new Error(
+    `warning: baseUrlPrefix must end with /, was: ${baseUrlPrefix}`,
+  );
+}
```

Comment:
do we need this if we defined the value for `baseUrlPrefix` ourselves up above?

Reply:
No, but it felt nice to ensure as an invariant in the code. I can flip it to a comment or assert it in a test.